src/libraries/Microsoft.PowerFx.Core/Texl/Builtins/ColorFadeT.cs

```diff
@@ -89,7 +89,7 @@ public override bool CheckInvocation(TexlBinding binding, TexlNode[] args, DType
                 fValid &= CheckNumericColumnType(type1, args[1], errors, ref nodeToCoercedTypeMap);
 
                 // Since the 1st arg is not a table, make a new table return type *[Result:c]
-                returnType = DType.CreateTable(new TypedName(DType.Color, OneColumnTableResultName));
+                returnType = DType.CreateTable(new TypedName(DType.Color, GetOneColumnTableResultName(binding.Features.HasFlag(Features.ConsistentOneColumnTableResult))));
```

Comment:
>GetOneColumnTableResultName(binding.Features.HasFlag(Features.ConsistentOneColumnTableResult)) [](http://example.com/codeflow?start=74&length=94)

Can we encapsulate this in a helper function so we can just say:

GetOneColumnTableResultName(binding)